NUnitTestMarsRover/HeadingTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using NUnit.Framework;
+using MarsRover;
+
+namespace NUnitTestMarsRover
+{
+    class HeadingTests
+    {
+        [Test]
+        public void NorthHeadingTurnRight_GivenARightTurnCommand_ReturnsEastHeading()
```

Comment:
These test names talk about commands, but commands aren't present in the tests; it's just method calls on the headings. You could simplify these names to something like

```
NorthHeading_TurnRight_ReturnsEastHeading
```

or 

```
TurnRight_GivenNorthHeading_ReturnsEastHeading
```

(this applies to all of the tests in this file).

Reply:
I will update those.